### ISSUE TITLE
http set header error

### ISSUE THREAD

**Issue by joehand** [datproject/dat#1023]
I am reporting:

- [x] a bug or unexpected behavior
- [ ] general feedback
- [ ] feature request
- [ ] security issue 

# Bug Report

Please give us details about your installation to assist you. Run `dat -v` to see the version of Dat you are using.

* Operating system: linux
* Node Version: v10
* Dat Version: 13.11

### Expected behavior

http serves files okay

### Actual behavior

Error:

```
hyperdrive-http/index.js:149
  res.setHeader('Content-Type', 'text/plain; charset=utf-8')
      ^

TypeError: res.setHeader is not a function
    at onerror (/home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/hyperdrive-http/index.js:149:7)
    at /home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/hyperdrive-http/index.js:67:31
    at /home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/hyperdrive-http/index.js:135:5
    at /home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/hyperdrive/index.js:537:5
    at f (/home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/once/once.js:25:25)
    at Class.<anonymous> (/home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/stream-collector/index.js:15:5)
    at Class.emit (events.js:187:15)
    at endReadableNT (/home/joe/.nvm/versions/node/v10.6.0/lib/node_modules/dat/node_modules/readable-stream/lib/_stream_readable.js:1010:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

**Comment by schnaitterm** [comment]
Confirm, I have the same issue.  Running on Fedora 28, node v10.7.0 and dat v13.11.3

**Comment by joehand** [comment]
Fixed in hyperdrive-http `4.3.3` https://github.com/datproject/hyperdrive-http/commit/0511fc895a819efd3dc0020b6006915fb816a3aa
